app/controllers/api/v1/pizzas_controller.rb

```diff
@@ -1,6 +1,27 @@
 class Api::V1::PizzasController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
   def show
     selected_pizza = Pizza.find(params[:id])
-    render json: { pizza: selected_pizza, brand: selected_pizza.brand }
+    render json: {pizza: selected_pizza, brand: selected_pizza.brand, reviews: selected_pizza.reviews} 
   end
+
+  def new
+    pizza = Pizza.find(params[:id])
+    review = Review.new
+  end
+
+  def create
+    pizza_id = params[:pizza].to_i
+    pizza = Pizza.find(params[:pizza])
+    review = Review.new(name: params[:name],body: params[:body],rating: params[:rating], pizza_id: pizza_id)
```

Comment:
please use strong params rather than pulling out all of the params individually

Reply:
in addition, line 17 is redunant with what you have at the end of line 16. I would just remove the pizza_id reference from the end of line 16 since you are going to refactor for strong params anyways